JavascriptFormatter.js

```diff
@@ -913,18 +956,19 @@ function array(value) {
     return JSON.stringify(value);
 }
 Equals.prototype.toString = function () {
-    return this.e1.toString() + " == " + this.e2.toString();
+    return this.e1.toString() + " === " + this.e2.toString();
 };
 Equals.prototype.assert = function () {
-    return "assert.equal(" + this.e1.toString() + ", " + this.e2.toString()
-        + ", 'Assertion error: Expected: " + this.e1.toString() + ", Actual: ' + "
-        + this.e2.toString() + ");";
+    var varA = getTempVarName();
+    var varB = getTempVarName();
+    return "var " + varA + " = " + this.e1.toString() + ";\n" + "var " + varB + " = " + this.e2.toString() + ";\n"
+        + "assert.equal(" + varA + ", " + varB + ", 'Assertion error: Expected: ' + " + varA + " + ', got: ' + " + varB + ");";
```

Comment:
the assert should tell user which string is wrong, not to tell them Expected 'var1', got: 'var2'